src/store/project.ts

```diff
@@ -211,6 +211,17 @@ export const projectStore: VoiceVoxStoreOptions<
             }
           }
 
+          if (baseVersionIsLow(appVersionList, [0, 10, 0])) {

```

Comment:
0.10.2で保存しちゃった人も救えるかもなので、こうすると良いかも？
```suggestion
          if (baseVersionIsLow(appVersionList, [0, 10, 3])) {

```

Reply:
と思ったけど0.10.2では存在するのでいらないとのこと！なるほどです